spec/features/admin/poll/polls_spec.rb

```diff
@@ -58,6 +58,8 @@
     fill_in "poll_name", with: "Upcoming poll"
     fill_in 'poll_starts_at', with: start_date.strftime("%d/%m/%Y")
     fill_in 'poll_ends_at', with: end_date.strftime("%d/%m/%Y")
+    fill_in 'poll_summary', with: "Upcoming poll's summary. This poll..."
+    fill_in 'poll_description', with: "Upcomming poll's description. This poll..."
```

Comment:
Missing the corresponding expectations for this input fields. 
The spec related to what a non-admin user sees, should also take these new fields into account.

Reply:
Missing image specs